test/extensions/quic_listeners/quiche/platform/epoll_test_impl.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+#define EpollTestImpl ::testing::Test
```

Comment:
nit: prefer type alias to macro:

namespace epoll_server {
 using EpollTestImpl = ::testing::Test;
 } // namespace epoll_server

Reply:
done